packages/oas-to-snippet/src/index.ts

```diff
@@ -45,20 +46,39 @@ export default async function oasToSnippet(
        */
       variableName?: string;
     };
+
+    /**
+     * `httpsnippet` plugins to extend snippet generation to.
+     *
+     */
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    plugins?: ClientPlugin<Record<string, any>>[];
```

Comment:
```suggestion
    plugins?: ClientPlugin<Record<string, unknown>>[];
```

Reply:
Have to use `any` here.

![Screen Shot 2023-11-03 at 10 03 00 AM](https://github.com/readmeio/oas/assets/33762/2fa30266-d865-45cf-9185-7315ef73e5f9)